runtime/libffi/module.xml

```diff
@@ -200,6 +223,9 @@
 			<vpath pattern="ffi64.c" path="x86" augmentObjects="true" type="relativepath">
 				<include-if condition="spec.flags.arch_x86 and spec.flags.J9VM_ENV_DATA64" />
 			</vpath>
+			<vpath pattern="darwin64.S" path="x86" augmentObjects="true" type="relativepath">
+				<include-if condition="spec.osx_x86.*" />
```

Comment:
Should this condition include `and spec.flags.J9VM_ENV_DATA64` since it seems only to apply to the 64-bit spec?